docs/projects/tad/reporting-standard/latest.md

```diff
@@ -323,9 +339,18 @@ legal_base:
 used_data: {system_used_data}                           # Optional. Description of the data used by the system.
 technical_design: {technical_design}                    # Optional. Description of the technical design of the system.
 external_providers:
-- {system_external_provider}                            # Optional. Reference to used external providers.
+- name: {name_external_provider}                        # Optional. Reference to used external providers.
+  version: {version_external_provider}                  # Optional. Version used of the external provider.
 references:
 - {reference_uri}                                       # Optional. Example: URI to codebase.
+interaction_details: {system_interaction_details}       # Optional. Example: "GPS modules for location tracking"
+version_requirements: {system_version_requirements}     # Optional. Example: "version2.1"
+deployment_variants: {system_deployment_variants}       # Optional. Example: "API"
+hardware_requirements: {system_hardware_requirements}   # Optional. Example: "8 GB RAM"
+product_markings: {system_product_markings}             # Optional. Example: where the AI system is a component of products, photographs or illustrations showing external features, the marking and internal layout of those products
```

Comment:
@ravimeijerrig the example here suggests `string` while it should be a list of strings. Can you change this?